bridge_adaptivity/module/static/module/js/module.js

```diff
@@ -224,23 +224,20 @@
         }
 
         // Launch URL fetching:
-        const launchUrlFetcher = new Clipboard('#launch-url-fetcher',);
         const launchRepetitionUrlFetcher = new Clipboard(
             '#launch-repetition-url-fetcher',
             {
                 text: trigger => {
                     return trigger.getAttribute('data-clipboard-text').replace('set_me_unique', (new Date().getTime()))
                 }
             });
-        const onLaunchURLCoppy = function (e) {
+        launchRepetitionUrlFetcher.on("success",  function (e) {
```

Comment:
why do you leave repetition instead of more short urlcopy?

Reply:
fix